### ISSUE TITLE
Different parent?

### ISSUE THREAD

**Issue by strarsis** [dollarshaveclub/stickybits#327]
It is possible to use a different parent element than the actual parent in DOM?

**Comment by yowainwright** [comment]
@strarsis 

No. If you would like to provide a PR, support would be considered. Initially, a feature like this would be really complex and cause issues for other features + weird support across browsers.

**Comment by yowainwright** [comment]
Readers, please give this issue a thumbs up if you would like this feature to be considered. 👍

**Comment by reefki** [comment]
+1 for this feature!

My site's header has 3 bars wrapped inside `<header>` tag and I only want to make the last bar sticky. Apparently, it's not possible to make the last header bar always on the top of the page without unwrapping those three bars out of `<header>` tag.

@yowainwright any other solution for this http://jsfiddle.net/Lkt26q70/1/ ?

**Comment by alejandroiglesias** [comment]
Did any progress happen on this?

**Comment by yowainwright** [comment]
@alejandroiglesias no. Would you like to work on it? 🙂
